romancal/resample/resample_utils.py

```diff
@@ -101,3 +103,32 @@ def make_output_wcs(
     )
 
     return wcs, pscale * 3600.0, pscale_ratio
+
+
+def add_var_sky_array(input_models: ModelLibrary):
```

Comment:
```suggestion
def compute_var_sky(model):
```
docstring will need to be updated.